plugins/modules/rds_instance.py

```diff
@@ -280,6 +280,14 @@
         description:
           - Specifies if the DB instance is a Multi-AZ deployment. Mutually exclusive with O(availability_zone).
         type: bool
+    multi_tenant:
+        description:
+          - Specifies whether to use the multi-tenant configuration or the single-tenant configuration (default).
+          - This parameter only applies to RDS for Oracle container database (CDB) engines.
+          - The DB engine that you specify in the request must support the multi-tenant configuration.
+          - If the multi-tenant configuration is enabled during creation of the DB instance, it cannot be modified later.
+        type: bool
+        version_added: 9.0.0
```

Comment:
I took a quick look at that, RDS got a refactored and that refactor hasn't been backported, releasing it October instead of November doesn't buy us much.